6_cifar10_cnn_coteaching/cuda_requirements.txt

```diff
@@ -0,0 +1,5 @@
+cleanlab
+numpy==1.21.6
+scikit_learn==1.0.2
+torch==1.10.1+cu113
+torchvision==0.11.2+cu113
```

Comment:
I wonder is the `cu113` part even really necessary in light of recent torch updates?

Reply:
Having the `+cu113` part was actually throwing me install errors, I think we can remove it